components/controls/__tests__/RadioToggle-test.tsx

```diff
@@ -29,10 +29,25 @@ it('renders', () => {
 it('calls onCheck', () => {
   const onCheck = jest.fn();
   const wrapper = shallow(getSample({ onCheck }));
-  change(wrapper.find('input[value="two"]'), 'two');
+  change(wrapper.find('input[value="second"]'), 'second');
   expect(onCheck).toBeCalledWith('two');
 });
 
+it('calls with number type', () => {
```

Comment:
`calls with number type` doesn't mean anything :smile: 
How about `handles numeric values`.
Also you should check that setting the value selects the expected option